pkg/operation/common/utils.go

```diff
@@ -227,6 +231,25 @@ func DeleteLoki(ctx context.Context, k8sClient client.Client, namespace string)
 	return nil
 }
 
+// DeleteShootNodeLoggingStack deletes all shoot resource of the shoot-node logging stack in the given namespace.
+func DeleteShootNodeLoggingStack(ctx context.Context, k8sClient client.Client, namespace string) error {
+	resources := []client.Object{
+		&extensionsv1beta1.Ingress{ObjectMeta: metav1.ObjectMeta{Name: "loki", Namespace: namespace}},
+		&corev1.Secret{ObjectMeta: metav1.ObjectMeta{Name: SecretNameLokiKubeRBACProxyKubeconfig, Namespace: namespace}},
+		&corev1.Secret{ObjectMeta: metav1.ObjectMeta{Name: LokiTLS, Namespace: namespace}},
+		&networkingv1.NetworkPolicy{ObjectMeta: metav1.ObjectMeta{Name: "allow-from-prometheus-to-loki-telegraf", Namespace: namespace}},
+		&corev1.ConfigMap{ObjectMeta: metav1.ObjectMeta{Name: "telegraf-config", Namespace: namespace}},
+	}
+
+	for _, resource := range resources {
+		if err := k8sClient.Delete(ctx, resource); client.IgnoreNotFound(err) != nil && !meta.IsNoMatchError(err) {
+			return err
+		}
+	}
+
+	return nil
```

Comment:
You can use `kutil.DeleteObjects` instead

Reply:
Thanks